global/scripts/MyelinMap_BC.sh

```diff
@@ -0,0 +1,180 @@
+log_Check_Env_Var CARET7DIR
+
+#display HCP Pipeline version
+log_Msg "Showing HCP Pipelines version"
+"${HCPPIPEDIR}"/show_version --short
+
+#display the parsed/default values
+opts_ShowValues
+
+# ------------------------------------------------------------------------------
+#  Main processing of script.
+# ------------------------------------------------------------------------------
+log_Msg "Starting main functionality"
+Caret7_Command=${CARET7DIR}/wb_command
+RegNameToUse=""
+if [[ "$RegName" != "" ]]; then
+	RegNameToUse=_${RegName}
+fi
+
+# default folders
+SubjFolder=${StudyFolder}/${Subject}
+AtlasSpaceFolder=${SubjFolder}/MNINonLinear
+T1wFolder=${SubjFolder}/T1w
+#NativeT1wFolder=${T1wFolder}/Native
+#NativeFolder=${AtlasSpaceFolder}/Native
+#DownSampleFolder=${AtlasSpaceFolder}/fsaverage_LR${LowResMesh}k
+#DownSampleT1wFolder=${T1wFolder}/fsaverage_LR${LowResMesh}k
+
+# convert upper case to lower case
+InputMesh=`echo "$InputMesh" | tr '[:upper:]' '[:lower:]'`
+OutputMesh=`echo "$OutputMesh" | tr '[:upper:]' '[:lower:]'`
+
+if [[ "$InputMesh" == "$OutputMesh" ]]; then
+	log_Err_Abort "the input mesh ${InputMesh} and the output mesh ${OutputMesh} shouldn't be the same!"
+fi
+
+case "$InputMesh" in
+	(native)
+		InputMeshString="native"
+		InputFolder=${AtlasSpaceFolder}/Native
+		InputT1wFolder=${T1wFolder}/Native
+		InputBCMapName="MyelinMap"
+		;;
+	(164)
+		InputMeshString=164k_fs_LR
+		InputFolder=${AtlasSpaceFolder}
+		InputT1wFolder=${AtlasSpaceFolder}
+		InputBCMapName="MyelinMap"
+		;;
+	(32)
+		InputMeshString=32k_fs_LR
+		InputFolder=${AtlasSpaceFolder}/fsaverage_LR32k
+		InputT1wFolder=${T1wFolder}/fsaverage_LR32k
+		InputBCMapName="atlas_MyelinMap"
+		;;
+	(*)
+        log_Err_Abort "unrecognized --input-mesh value '$InputMesh', valid options are native, 32, 164"
+        ;;
+esac
+
+case "$OutputMesh" in
+	(native)
+		OutputMeshString="native"
+		OutputFolder=${AtlasSpaceFolder}/Native
+		OutputT1wFolder=${T1wFolder}/Native
+		OutputBCMapName="MyelinMap"
+		;;
+	(164)
+		OutputMeshString=164k_fs_LR
+		OutputFolder=${AtlasSpaceFolder}
+		OutputT1wFolder=${AtlasSpaceFolder}
+		OutputBCMapName="MyelinMap"
+		;;
+	(32)
+		OutputMeshString=32k_fs_LR
+		OutputFolder=${AtlasSpaceFolder}/fsaverage_LR32k
+		OutputT1wFolder=${T1wFolder}/fsaverage_LR32k
+		OutputBCMapName="atlas_MyelinMap"
+		;;
+	(*)
+        log_Err_Abort "unrecognized --output-mesh value '$OutputMesh', valid options are native, 32, 164"
+        ;;
+esac
+
+IndividualMap=${InputFolder}/${Subject}.MyelinMap${RegNameToUse}.${InputMeshString}.dscalar.nii
+log_File_Must_Exist "${IndividualMap}"
+ReferenceMap=${InputFolder}/${Subject}.${InputBCMapName}_BC.${InputMeshString}.dscalar.nii
+log_File_Must_Exist "${ReferenceMap}"
+ReferenceMapOld=${InputFolder}/${Subject}.${InputBCMapName}_oldBC.${InputMeshString}.dscalar.nii
+ReferenceMapToUse=${ReferenceMap}
+
+# match the reference map's mean with the individual map's mean
+if ((UseIndMeanBool)); then
```

Comment:
I think you are missing the `UseIndMeanBool=$(opts_StringToBool "$UseIndMean")` step for this.

Reply:
Oops. My bad.